src/models/index.js

```diff
@@ -17,11 +17,21 @@ const sequelize = new Sequelize(config.get('DATABASE_URL'), {
   logging: false
 })
 
+// add customized toJSON method to a model
+const addMethodOfToJSON = (model) => {
+  model.prototype.toJSON = function () {
```

Comment:
@imcaizheng I have some concerns regarding the naming of this method. Sequalize model already has method `toJSON` so this would override it which could potentially lead to confusion. Can we use a different name for this? Perhaps something like `getCleared` or `getCleanObject` to indicate that this method return some cleaned/prepared data of the document.

Reply:
@imcaizheng before making any fix to this there is one idea. Do I understand it right, that if instead of manual control of `deletedAt` we are utilizing built-in `paranoid` mode of sequalize, then `deletedAt` would NOT be returned automatically?

i. e. if we implement https://github.com/topcoder-platform/taas-apis/issues/132, then we don't have to implement a custom method `getCleared` and `deletedAt` would not be returned by the standard `toJSON` or `get`?

Reply:
@maxceem You are indeed right. After we implement #132, we don't need to manually remove the `deletedAt` field. Currently this `toJSON` function can be served as a temporary fix.

Reply:
@imcaizheng ok, great. 

Let's implement this together with  #132 so we can test how both fixes work together. You may implement #132 in the same branch as this one `imcaizheng:post-event-with-payload-from-db-data` so all the changes would be included into this PR.

Just in case, there are some merge conflicts after merging another PR.